### PR TITLE
Uppercase the me in README

### PR DESCRIPTION
**Pull request by robphoenix**
## Description

I believe uppercasing README is a known convention, though if I'm mistaking personal preference here then please do ignore me and close this PR. 😃

### CODE REVIEW

File: docs/docs/using-gatsby-image.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ So this is all very nice and it’s far better to be able to use this from NPM v
 
 ### References:
 
-- [Plugin READme file](/packages/gatsby-image/)
+- [Plugin README file](/packages/gatsby-image/)
```
Comment (muescha) [inline docs/docs/using-gatsby-image.md]:
```suggestion
- [Gatsby image plugin README file](/packages/gatsby-image/)
```

**Comment by robphoenix** [comment]
Good catch @muescha, thanks. I've updated the PR. Do you need me to rebase the commits into one commit?

**Comment by LekoArts** [comment]
@robphoenix 	The PRs get squashed into one commit on merge, so you can do as many as commits as you like 👍

**Comment by robphoenix** [comment]
Ah super 👌 thanks

**Comment by wardpeet** [comment]
@robphoenix thanks for making this edits, if you go through the entire code base you'll see more of these READme cases. Any chance you can update them too?

I'm cc'ing @gatsbyjs/ecosystem just to make sure it's a necessary change. (you might want to hold of on changing all READme into README)

**Comment by robphoenix** [comment]
Yeah, sure, I could get that done sometime today if there's no reason not to 👍

**Pull request merged**